### PR TITLE
Change ci machine type from e1 to f1

### PR DESCRIPTION
**Pull request by d-goncalves**

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name: "Vaultenv CI Pipeline"
 
 agent:
   machine:
-    type: "e1-standard-4"
+    type: "f1-standard-4"
```
Comment (bertptrs) [inline .semaphore/semaphore.yml:7]: Shouldn't this be

```suggestion
    type: "f1-standard-2"
```

Comment (d-goncalves) [reply .semaphore/semaphore.yml:7]: The idea was to keep the same amount of cores per run. But if you think 2 cores should be enough I'll ship that

Comment (bertptrs) [reply .semaphore/semaphore.yml:7]: Then I misexplained, my intention (and Semaphore's recommendation) was to substitute 1 f1 core for every 2 e2 cores, on the grounds that the f1 cores still outperformed at that conversion rate.

**Comment by d-goncalves** [comment]
@OpsBotPrime merge on friday

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @d-goncalves, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as c61036674d3a385523b38e5b30105148e91f6cfd, waiting for CI …

**Comment by OpsBotPrime** [comment]
[CI job :yellow_circle:](https://channable.semaphoreci.com/workflows/6966b271-cf8f-40b6-9c87-42f6bccfc1d8?pipeline_id=c19e4264-b9f2-437f-8417-fe0b7a28b648) started.

**Pull request merged**